proto/contimage/contimage.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contimage";
+
+package datadog.contimage;
+
+// ContainerImagePayload represents the main container image payload
+message ContainerImagePayload {
+    string                  version = 1;  // version represents the ContainerImagePayload message version
+    string                  host    = 2;  // host contains the hostname
+    repeated ContainerImage images  = 3;  // Images metadata
+    // repeated VulneratibilityReport reports = 4; // vulneratibility reports
+}
+
+// ContainerImage contains the details of a container image
+message ContainerImage {
+    string          id           = 1;
+    string          registry     = 2;
+    string          short_name   = 3;
```

Comment:
Don't we also need a `name` field in addition to `short_name` and `registry`?

Reply:
I’ve added `name` in addition to `registry` and `short_name`: https://github.com/DataDog/agent-payload/pull/196/commits/f4f9622bf37230f88ac3e92a9a18e808925989b4.